doc/desiInstall.rst

```diff
@@ -224,6 +222,9 @@ src
     :command:`desiInstall` will attempt to execute :command:`make -C src all`.  This build type
     *is* mutually exclusive with 'make', but is not mutually exclusive with
```

Comment:
This contradicts the statement above that the build types are mutually exclusive.

Reply:
Thanks for catching that; I corrected it.